lib/plenario_web/controllers/api/aot_controller.ex

```diff
@@ -74,10 +78,19 @@ defmodule PlenarioWeb.Api.AotController do
       case Map.get(conn.params, "bbox") do
         nil -> metas
         value ->
-          bbox = parse_bbox(value)
+          bbox = parse_bbox(value, conn)
           from m in metas, where: st_intersects(m.bbox, ^bbox)
       end
 
+    try do
+      Repo.all(metas)
```

Comment:
Why make the database do this expensive query when we're not using the results other than to validate the JSON isn't gibberish?

Reply:
Woops, this is supposed to be `Repo.one` - we have to `try` the query in order to catch the `postgrex` geometry error.

Reply:
But then it's still running a really big query and truncating the results to the first one...